edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/ExperimentalFeatures.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Service
+{
+    using Microsoft.Extensions.Configuration;
+
+    public class ExperimentalFeatures
+    {
+        public ExperimentalFeatures(bool enabled, bool disableCloudSubscriptions, bool disableConnectivityCheck)
+        {
```

Comment:
private
I see this is used in the test, but a mock can be provided instead

Reply:
Any reason to keep this private? It is not a singleton.. 
I see the Init/Create more as a helper than a factory method here.

Reply:
At the moment this only has 3 parameters, but with new features it will be a few bool parameters which can be difficult to make sure they are provided in the right order, using Create with a configuration solves the problem.